From a322051c7d83d54cf495d987d97a8a80469a8677 Mon Sep 17 00:00:00 2001
From: Papa Gruff <sickofmonkeys@gmail.com>
Date: Mon, 17 Sep 2018 11:03:39 -0400
Subject: [PATCH] Only enable presidential CMAS alerts if user is a monkey

* This patch is in response to FEMA's announcement of the nationwide test
  of the EAS on Thursday, September 20, 2018. In case this is not obvious,
  this patch is a joke, please do not actually disable presidential alerts.

Change-Id: Ib982ee2ab4e91667540341a7c274f1cae58473c7
---
 .../cellbroadcastreceiver/CellBroadcastAlertService.java      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
index 38750030..d398116b 100644
--- a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
+++ b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
@@ -442,8 +442,8 @@ public class CellBroadcastAlertService extends Service {
         // CMAS warning types
         if (CellBroadcastChannelManager.checkCellBroadcastChannelRange(subId,
                 channel, R.array.cmas_presidential_alerts_channels_range_strings, this)) {
-            // always enabled
-            return true;
+            // only enabled if user is a monkey
+            return ActivityManager.isUserAMonkey();
         }
         if (CellBroadcastChannelManager.checkCellBroadcastChannelRange(subId,
                 channel, R.array.cmas_alert_extreme_channels_range_strings, this)) {
-- 
2.17.1

